sky/services/oknet/src/org/domokit/oknet/UrlLoaderImpl.java

```diff
@@ -248,9 +249,13 @@ public void onResponse(Response response) {
                     MediaType mediaType = body.contentType();
                     if (mediaType != null) {
                         urlResponse.mimeType = mediaType.type() + "/" + mediaType.subtype();
-                        Charset charset = mediaType.charset();
-                        if (charset != null) {
-                            urlResponse.charset = charset.displayName();
+                        try {
+                          Charset charset = mediaType.charset();
+                          if (charset != null) {
+                              urlResponse.charset = charset.displayName();
+                          }
+                        } catch (UnsupportedCharsetException e) {
+                          Log.e(TAG, "Unsupported charset", e);
```

Comment:
In this case, it's not really a fatal error, the charset on the urlResponse is null but the image renders fine.